src/common/storage/backends/s3/metadata.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "S3 Storage",
+  "configStructure": [
+    {
+      "name": "endpoint",
+      "displayName": "S3 Server URL",
+      "description": "URL for S3 Server",
```

Comment:
It might be nice to give an example here. This description doesn't really tell us anything more than the display name.

Reply:
An example for this has been added to the metadata.